portal/app/templates/main/index.html

```diff
@@ -128,13 +75,50 @@ <h2>Train</h2>
     </div>
   </div>
 </div>
+<!-- <div class="container">
```

Comment:
Why is this part of the HTML commented, if it is not needed anymore, please remove it